DATA/production/configurations/2022/LHC22f/apass1/setenv_extra.sh

```diff
@@ -100,7 +118,12 @@ DEBRIS=""
 if [[ $BEAMTYPE == "pp" ]]; then
   DEBRIS="pvertexer.maxChi2TZDebris=10"
```

Comment:
actually, the `pvertexer.maxChi2TZDebris` is already set to 10 / 2000 for pp / pbpb in the dpl-workflow.sh